cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -300,7 +301,7 @@ def certificate_activation_handler(request, course_key_string):
     # Only global staff (PMs) are able to activate/deactivate certificate configuration
     if not GlobalStaff().has_user(request.user):
         raise PermissionDenied()
-    course_key = CourseKey.from_string(course_key_string)
+    course_key = from_string_or_404(course_key_string)
```

Comment:
`from_string_or_404` this method is actually building `course_key`?
If yes, I wish it had name like `course_key_from_string_or_404`. But I am open.